### PR TITLE
Added the tuned importer (from ADIF) to QSO-API as well

### PR DESCRIPTION
**Pull request by int2001**
API is now capable of Bulk-Import

### CODE REVIEW

File: application/controllers/Api.php [hunk 1]
```diff
@@ -198,49 +204,33 @@ function qso($dryrun = false) {
...
 					}
+					if(count($record) == 0) {
+						break;
+					};
+					array_push($alladif,$record);
+					$return_count++;
+				};
+				$record='';	// free memory
+				gc_collect_cycles();
+				log_message("Error","Start");
```
Comment (phl0) [inline application/controllers/Api.php:226]: Debug leftover?

**Review by phl0 — CHANGES_REQUESTED** [review]
Debug code forgotten to remove?

File: application/controllers/Api.php (hunk 1 above)
Comment (int2001) [reply application/controllers/Api.php]: tnx. removed

File: assets/json/dok.txt
Comment (phl0) [inline assets/json/dok.txt:1]: Should this and the updates of clublog and lotw be included here?!

**Review by phl0 — CHANGES_REQUESTED** [review]
Changes in dok.txt, clublog_scp.txt and lotw_users.csv accidentially slipped into commit 1104f30bd0187bc1cd1739932abb9ffc6daf7aab. I think this should be reverted / committed in a separate PR.

File: assets/json/dok.txt
Comment (int2001) [reply assets/json/dok.txt:1]: Sri - removed from tracking now

**Review by phl0 — APPROVED** [review]
Importing multiple QSOs with a single API call works.

**Pull request merged**